### PR TITLE
Updated zookeeper dashboard changes and screenshot

### PR DESCRIPTION
**Pull request by RamanaReddy8801**
**Ticket**: https://issues.newrelic.com/browse/NR-145557
**Description**: Updated zookeeper dashboard changes and screenshot

# Summary

A concise description of the changes being introduced. Please review the pre-merge checklist section to validate this pull request is ready for review and merge. **If it is not ready, please mark the pull request as a draft.**

_The owners of this repo are not experts in the subject matter of the quickstarts. We review for the quickstart to be functional and for security risks. If you are seeking feedback on the content of the quickstart, please seek out a subject matter expert. If you are not an internal NR contributor, we can do our best to connect you with a content reviewer._

## Pre merge checklist

- [ ] Did you check you NRQL syntax? - Does it work?
- [ ] Did you include a Data source and Documentation reference?
- [ ] Are all documentation links publicly accessible?
- [ ] Did you check your descriptive content for [voice and tone](https://docs.newrelic.com/docs/style-guide/writing-strategies/voice-strategies-docs-sound-new-relic/)?
- [ ] Did you check your descriptive content for spelling and grammar errors?
- [ ] Did you review your content with a subject matter expert? (e.g. a Browser agent quickstart is reviewed with a member of the Browser Agent team)

### Dashboards

- [ ] Does the PR contain a screenshot for each of your dashboards?
- [ ] Do your screenshots show data?
- [ ] Has the [sanitization script](https://github.com/newrelic/newrelic-quickstarts/blob/main/CONTRIBUTING.md#dashboards) been run on each dashboard?

### Alerts

- [ ] Did you check that your alerts actually work?

### CODE REVIEW

**Review by mdumpati — DISMISSED** [review]
LGTM

**Review by seemantk — APPROVED** [review]
lgtm, thank you @RamanaReddy8801

File: dashboards/apache-zooKeeper/dashboard.json [hunk 1]
```diff
@@ -203,7 +212,7 @@
...
+            "text": "**About**\n\nInstrument your application with New Relic - [Add Data](https://one.newrelic.com/catalog-pack-details?state=572ff87b-6e31-c7d4-d457-d53b8ffd0924)\n\nUnable to find data in your dashboard? - [Troubleshoot here](\nhttps://docs.newrelic.com/docs/infrastructure/host-integrations/host-integrations-list/flex-integration-tool-build-your-own-integration/)\n\n\n[Please rate this dashboard](https://docs.google.com/forms/d/e/1FAIpQLSclR38J8WbbB2J1tHnllKUkzWZkJhf4SrJGyavpMd4t82NjnQ/viewform?usp=pp_url&entry.1615922415=ZooKeeper&entry.358368110=https://onenr.io/0MR280PZJQY) here and let us know how we can improve it for you."
```
Comment (pkosanam) [inline dashboards/apache-zooKeeper/dashboard.json:215]:
```suggestion
            "text": "**About**\n\nInstrument your application with New Relic - [Add Data](https://one.newrelic.com/catalog-pack-details?state=572ff87b-6e31-c7d4-d457-d53b8ffd0924)\n\nUnable to find data in your dashboard? - [Troubleshoot here](\nhttps://docs.newrelic.com/docs/infrastructure/host-integrations/host-integrations-list/flex-integration-tool-build-your-own-integration/)\n\n\n[Please rate this dashboard](https://docs.google.com/forms/d/e/1FAIpQLSclR38J8WbbB2J1tHnllKUkzWZkJhf4SrJGyavpMd4t82NjnQ/viewform?usp=pp_url&entry.1615922415=ApacheZooKeeper&entry.358368110=https://onenr.io/0MR280PZJQY) here and let us know how we can improve it for you."
```

**Review by pkosanam — CHANGES_REQUESTED** [review]

**Review by mdumpati — APPROVED** [review]
LGTM

**Review by pkosanam — APPROVED** [review]
LGTM

**Review by seemantk — APPROVED** [review]

**Pull request merged**